app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md

```diff
@@ -23,15 +23,27 @@ SKE courses can help when you have:
 - a different but related degree to the subject you want to teach
 - an A level but not a degree in the subject
 - an unrelated degree but relevant professional experience
-- a languages degree but need to improve your knowledge of a language you’ll teach
+- studied for a languages degree, but need to develop a second language to an acceptable level for teaching in schools 
 - not used your subject knowledge for a while
 
-When you apply for teacher training, the school or university will help you establish whether you need to improve your subject knowledge.
+When you apply for teacher training, the school or university will help you establish whether you need to improve your subject knowledge. They may decide to make completing a SKE course a condition of your offer to train with them.
 
-Completing a course may be a condition of your offer to train.
+SKE courses are currently offered in 9 secondary subjects:  
 
-They’re typically offered for subjects where there’s a particular demand for teachers.
+- mathematics (including primary mathematics) 
+- English 
+- physics 
+- chemistry 
+- biology 
+- computing 
+- languages 
+- design and technology  
+- religious education
 
-Courses are available across England and are often online. You can do them before or alongside your teacher training. You may also be able to study part-time.
+SKE course lengths can range from 8 to 28 weeks and are usually online courses, but some are classroom based and some, a mixture of both. You can choose the provider you wish to complete your SKE course with.
 
-[Find out if there's a subject knowledge enhancement course for your subject](https://www.gov.uk/government/publications/subject-knowledge-enhancement-course-directory/subject-knowledge-enhancement-ske-course-directory).
+Courses are available across England. Most candidates complete a SKE course before starting their teacher training year, but it’s sometimes possible to complete it alongside your ITT course. You may also be able to study part-time.
+
+You may also be eligible to receive the SKE participant bursary, contact your ITT and SKE provider for more information. For academic year 2022/23, the participant bursary funding is £175 per week.  
+
+[Find out if there's a SKE course for your subject](https://www.gov.uk/government/publications/subject-knowledge-enhancement-course-directory/subject-knowledge-enhancement-ske-course-directory).
```

Comment:
```suggestion
[Find out if there's a subject knowledge enhancement (SKE) course for your subject](https://www.gov.uk/government/publications/subject-knowledge-enhancement-course-directory/subject-knowledge-enhancement-ske-course-directory).
```

I think it would be better to spell out what SKE means in this link - better for accessibility for screen reader users for example, if they have skipped over the para where this acronym is explained. Also - VoiceOver doesn't pronounce 'SKE' is a very clear way

Reply:
Happy to make this change.